### PR TITLE
change the specific object

### PR DESCRIPTION
**Pull request by natezhengbne**
I extracted the specific object outside the "showDialog".
I think maybe it is more flexible.
How about it ?

### CODE REVIEW

**Comment by xsahil03x** [comment]
@natezhengbne can you solve the build error.
"The current Dart SDK version is 2.1.0-dev.9.4.flutter-f9ebf21297.
Because giffy_dialog requires SDK version >=2.1.0 <3.0.0, version solving failed"

File: pubspec.yaml [hunk 1]
```diff
@@ -5,13 +5,12 @@ author: Sahil Kumar <xdsahil@gmail.com>
 homepage: https://github.com/xsahil03x/giffy_dialog
 
 environment:
-  sdk: ">=2.0.0-dev.68.0 <3.0.0"
+  sdk: ">=2.1.0 <3.0.0"
```
Comment (xsahil03x) [inline pubspec.yaml]: Do not change the sdk version as we are using stable channel.

**Comment by Purus** [comment]
Has this PR been reviewed? Please do the needful to review and merge.

**Comment by aagarwal1012** [comment]
This PR is good to merge, really provides flexibility, only you have to resolve that `flutter analyze` issue
```
Unused import:
          'package:cached_network_image/cached_network_image.dart' •
          lib/src/network.dart:1:8 • unused_import```

**Comment by xsahil03x** [comment]
@natezhengbne thanks for the contribution.

**Comment by xsahil03x** [comment]
@all-contributors please add @Kawaeee for code

**Comment by xsahil03x** [comment]
@all-contributors please add @natezhengbne  for code, example

**Pull request merged**